python_modules/dagster/dagster/_core/pipes/utils.py

```diff
@@ -251,19 +252,28 @@ class PipesThreadedMessageReader(PipesMessageReader):
     """
 
     interval: float
-    log_readers: Sequence["PipesLogReader"]
+    log_readers: Dict[str, "PipesLogReader"]
     opened_payload: Optional[PipesOpenedData]
     launched_payload: Optional[PipesLaunchedData]
 
     def __init__(
         self,
         interval: float = 10,
-        log_readers: Optional[Sequence["PipesLogReader"]] = None,
+        log_readers: Optional[Mapping[str, "PipesLogReader"]] = None,
     ):
         self.interval = interval
-        self.log_readers = check.opt_sequence_param(
-            log_readers, "log_readers", of_type=PipesLogReader
-        )
+        self.counter = 1
```

Comment:
Why do we set `counter` for `ThreadedMessageReader`? It's not used is it?

Reply:
leaked from the original pre-split PR